### ISSUE TITLE
The back end javascript file is included in the front end

### ISSUE THREAD

**Issue by xchs** [Paratron/contao-mod-mobile-visibility#1]
First of all, really helpful extension! Thanks a lot for that. I'd love to see it someday in the official Contao extension repository as well :)

Recently, I came across an issue regarding the [`backend.js`](https://github.com/Paratron/contao-mod-mobile-visibility/blob/master/assets/backend.js) javascript file being included both in the back end and the front end. I guess the script is only used in the **back end** to toggle the mobile/desktop visibility of an element, isn't it?
- [`config/config.php`](https://github.com/Paratron/contao-mod-mobile-visibility/blob/master/config/config.php#L13):

``` php
if (TL_MODE == 'BE')
{
  $GLOBALS['TL_JAVASCRIPT'][] = 'system/modules/kiss_mobile-visibility/assets/backend.js';
}
```

**Comment by Paratron** [comment]
Hey,
I htink applying it only in backend mode is okay.
Did you test this to be working?

**Comment by xchs** [comment]
Yes, it worked quite well. However, with my above solution, the script is included _everywhere_ in the back end, even where it is actually not really necessary. Perhaps it would be even better to embed the script only for the **site structure**, **article module**, and **content elements**, respectively.

**Comment by stefanschleich** [comment]
@Paratron I'd like to talk to you about the development of your module. Could you send me an email to stefan . plusx . de so i get your contact details? Thanks :)

**Comment by Paratron** [comment]
What do you want to know? You can mail me at hello@wearekiss.com (and I speak german)

**Comment by zonky2** [comment]
der Bug ist immer noch an Board :-(
